infra/examples/aws-zoom-slack/variables.tf

```
@@ -0,0 +1,34 @@
+variable "aws_account_id" {
+  type        = string
+  description = "id of aws account in which to provision your AWS infra"
+  validation {
+    condition     = can(regex("^\\d{12}$", var.aws_account_id))
+    error_message = "The aws_account_id value should be 12-digit numeric string."
+  }
+}
+
```

I'll add a sensitive variable to pass the access token:

```suggestion
variable "access_token" {
  type         = string
  sensitive  = true
  description = "value of the access token; it will be stored in secret manager/vault"
}
```